models/models.js

```diff
@@ -29,6 +27,17 @@ exports.selectArticles = (article_id) => {
     });
 };
 
+//GET API ARTICLES
+exports.selectArticles = () => {
+  return db
+    .query(
+      "SELECT articles.*, CAST(COUNT(comments.article_id)AS INT) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at ASC"
```

Comment:
your db query should change as a result of arguments passed into this function (drawn from req.params)
you'll also need to use <array variable>.includes to protect against SQL injection.

I'd suggest having a look through the notes for this one